src/main/scala/finalmp/controllers/PlayerController.scala

```diff
@@ -0,0 +1,20 @@
+package finalmp.controllers
+
+import finalmp.models._
+import java.util.UUID
+
+final case class PlayerController() {
+  val players = List[Player]()
+
+  def addPlayer(): Player = {
+    val newPlayer = Player(
+      PlayerId(UUID.randomUUID().toString),
+      PlayerName("Player_" + (players.length + 1).toString),
+      PlayerPosition(0, 0, 0)
+    )
+  
+    players.concat(List(newPlayer))
```

Comment:
You are aware that this will not update `players`, right ?

Reply:
Nope 😬
But I think for this MR it is currently okay. The implementation should be changed anyway.